src/g_editor.c

```diff
@@ -1909,6 +1945,12 @@ static t_editor *editor_new(t_glist *owner)
     sprintf(buf, ".x%lx", (t_int)owner);
     x->e_guiconnect = guiconnect_new(&owner->gl_pd, gensym(buf));
     x->e_clock = 0;
+    t_editor_private *private = 0;
+    private = getbytes(sizeof(*private));
+    x->e_privatedata = private;
+#ifdef HAVE_KEYBOARDNAV
+    initialize_kbdnav(&private->kbdnav);
```

Comment:
If `kbdnav` was a pointer (see https://github.com/pure-data/pure-data/pull/869/files#r367295268), this would have to be something like `kbdnav_new `which also allocates the memory for the `kbdnav` struct itself.

Reply:
I've just moved some implementations to `g_kbdnav.c` so i can get everything working that way.

After that i'll work in making this a pointer.

Reply:
fixed b768c3b1b916e3dce649a30f3ca9209d630fdc18